tests/step_implementers/automated_governance/test_automated_goverance.py

```diff
@@ -0,0 +1,123 @@
+import json
+import os
+import re
+# import sh
+import hashlib
+from pathlib import Path
+
+import sh
+from testfixtures import TempDirectory
+from ploigos_step_runner.step_implementers.automated_governance import Rekor
+from ploigos_step_runner.step_result import StepResult
+from tests.helpers.base_step_implementer_test_case import \
+    BaseStepImplementerTestCase
+from unittest.mock import patch
+
+
+class TestStepImplementerAutomatedGovernanceRekor(BaseStepImplementerTestCase):
+    def create_step_implementer(
+            self,
+            step_config={},
+            results_dir_path='',
+            results_file_name='',
+            work_dir_path=''
+    ):
+        return self.create_given_step_implementer(
+            step_implementer=Rekor,
+            step_config=step_config,
+            step_name='automated_governance',
+            implementer='Rekor',
+            results_dir_path=results_dir_path,
```

Comment:
this `results_dir_path` and `results_file_name` is the "old way" before the workflow reporting changes got recently merged. you can check out the "new way" here: https://github.com/ploigos/ploigos-step-runner/blob/main/tests/step_implementers/sign_container_image/test_podman_sign.py#L75-L90

I assume this is just because this change was started before the other changes were merged.